DOC: update references to pyansys/actions (#211)
pyansys/dev-guide: doc/source/how-to/packaging.rst

Dependabot version updates are performed by checking a ``dependabot.yml``
configuration file into your repository. In this file, one should specify the
location of the project's requirement files, so that Dependabot knows where to
look. The following code snippets show the required configuration for
Dependabot according to the type of file in which the dependencies are

> specified:

Reviewer: Should we talk as well about allowing dependabot to update your actions? I can't seem to find this info anywhere in the dev-guide, but I may be mistaken.

Author: I briefly cited the capabilities of dependabot for updating actions versions. On top of that, code snippets were updated with the required code for enabling such feature.

Dependabot version updates are performed by checking a ``dependabot.yml``

> configuration file into your repository. In this file, one should specify the

Reviewer 2: ```suggestion
configuration file into your repository. In this file, you should specify the
```

> location of the project's requirement files, so that Dependabot knows where to

Reviewer 2: ```suggestion
location of the project's requirement files so that Dependabot knows where to
```

> look. On top of that, Dependabot is also capable of updating ``GitHub actions``

Reviewer 2: ```suggestion
look. On top of that, Dependabot is capable of updating ``GitHub actions``
```

> versions.

Reviewer 2: Does GitHub actions need to be a literal string. It seems a common term and not something that is code-specific. Also, I'd recommend "...capable of updating versions for GitHub actions."

Author: You are right Kathy. No need to have this sort of highlight on the GitHub actions word.

> In order to enable security updates and notifications for your repository, please go to

Reviewer 2: ```suggestion
T enable security updates and notifications for your repository, go to
```
